mne_bids/read.py

```diff
@@ -76,6 +77,43 @@ def _read_raw(raw_fpath, electrode=None, hsp=None, hpi=None, config=None,
     return raw
 
 
+def _handle_participants_reading(participants_fname, raw,
+                                 subject, verbose=None):
+    participants_tsv = _from_tsv(participants_fname)
+    subjects = participants_tsv['participant_id']
+    row_ind = subjects.index(subject)
+
+    # map age to a random date of that year
+    age = participants_tsv['age'][row_ind]
+    meas_date = raw.info['meas_date']
+
+    # set data from participants tsv into subject_info
+    for infokey in participants_tsv.keys():
```

Comment:
```suggestion
    for infokey, infovalue in participants_tsv.items():
```

then below you can avoid all the `articipants_tsv[infokey]` and replace them with infovalue